pyerrors/input/__init__.py

```diff
@@ -4,3 +4,4 @@
 from . import misc
 from . import openQCD
 from . import sfcf
+from . import utils
```

Comment:
I think importing utils in the individual modules should be sufficient, the corresponding functions do not need to be available in the global namespace.

Reply:
Okay, fair point